cmd/kubeadm/app/util/certs/util.go

```diff
@@ -26,13 +26,16 @@ import (
 
 	certutil "k8s.io/client-go/util/cert"
 	"k8s.io/client-go/util/keyutil"
+	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
```

Comment:
CertConfig should be defined here, so that the kubeadm API does not have to be imported in the utils.
ultimately it is an utility structure similarly to the wrapped one from client-go.

Reply:
Thanks! Will do.
This should fix the failing typecheck test as well.

Reply:
@neolit123 I had to move `CertConfig` to `pkiutil`, otherwise there would be another import loop (`certs/util` ->  `pkiutil` -> `certs/util`).

Perhaps it makes sense to merge `certs/util` into `pkiutil` at some point.

Reply:
i think what i meant was "util/pkiutil" and not "util/certs".
it seems like the better location.